From aa0d736b079446fd549b7c45e511fccf9456e1aa Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 10 May 2022 08:59:32 +0800
Subject: [PATCH] atf-s32g: Set CAN clock to FOSC and 40MHz

Because of automotive requirements the CAN clock shall not have an PLL
in-between. The flexcan driver is expecting a 40Mhz clock therefore.

This patch bases on 0001-Set-CAN-clock-to-FOSC-and-40MHz.patch which is
at meta-aptiv/recipes-aptiv/u-boot-s32/files.

Upstream-Status: Pending

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 fdts/fsl-s32-gen1.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 9d945b19f..eda8bfd16 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -392,7 +392,7 @@
 				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
 				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
 				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI2>,
+				<&plat_clks S32GEN1_CLK_FXOSC>,
 				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI3>,
 				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI4>,
 				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI5>,
@@ -415,7 +415,7 @@
 				<80000000>,
 				<40000000>,
 				<40000000>,
-				<80000000>,
+				<40000000>,
 				<125000000>,
 				<200000000>,
 				<100000000>,
-- 
2.25.1

